### PR TITLE
Rename `DidUri` to `Did`

### LINKED ISSUE #399: Rename `DidUri` to `Did`
Context here: https://github.com/TBD54566975/web5-spec/issues/112#issuecomment-1924691567

To Do:
- Rename `DidUri` utility class to `Did`

### PR DESCRIPTION
**Pull request by frankhinek**
Rename the `DidUri` to `Did` per https://github.com/TBD54566975/web5-spec/issues/112#issuecomment-1924691567

closes #399

### CODE REVIEW

File: packages/dids/src/did.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export class DidUri {
   params?: Record<string, string>;
 
   /**
-   * Constructs a new `DidUri` instance.
+   * Constructs a new `DidUri` instance from individual components.
```
Comment (LiranCohen) [inline packages/dids/src/did.ts:97]:
```suggestion
   * Constructs a new `Did` instance from individual components.
```

**Review by LiranCohen — COMMENTED** [review]
Just one missed comment.

File: packages/dids/src/did.ts (hunk 1 above)
Comment (frankhinek) [reply packages/dids/src/did.ts]: Thanks @LiranCohen.  Went back and found 3 that were missed so pushed up 85ec434

**Review by LiranCohen — APPROVED** [review]
🚀

**Review by mistermoe — APPROVED** [review]

**Pull request merged**